microsoft-edge/web-platform/user-agent-string.md

```diff
@@ -9,23 +9,28 @@ ms.prod: microsoft-edge
 
-## User Agent Overrides
-Sometimes, a website doesn't recognize the updated version of Microsoft Edge's UA. As a result, a set of that website's features may not work correctly. When we are made aware of these types of issues, our team will reach out to website owners and let them know about the updated UA. 
+## User Agent Overrides  
 
-These sites often need some time to update and test their UA detection logic to address the issues that we report to them. In these cases, we use a Microsoft-provided list of UA overrides in our Beta channel to maximize compatibility for users who access these sites. These overrides specify new UA values that Microsoft Edge should send instead of its default UA to specific sites. You can view the list of UA overrides that are currently being applied by navigating to `edge://compat/useragent` in Microsoft Edge Beta.
+Sometimes, a website does not recognize the updated version of the Microsoft Edge UA.  As a result, a set of the features of that website may not work correctly.  When Microsoft is notified about these types of issues, website owners are contacted and informed about the updated UA.  
 
-Our Canary and Dev channels do not currently receive UA overrides since we wanted to provide an environment where web developers could easily reproduce issues on their sites that are caused by the default Microsoft Edge UA. If for some reason you require the ability to disable UA overrides in Microsoft Edge Beta, you can run `msedge.exe` using the following command line argument: `--disable-domain-action-user-agent-override`
\ No newline at end of file
+The sites often need some time to update and test the UA detection logic to address the issues that Microsoft reports to site owners.  In these cases, Microsoft uses a list of UA overrides in our Beta channel to maximize compatibility for users who access these sites.  The overrides specify new UA values that Microsoft Edge should send instead of the default UA for specific sites.  You are able to view the list of UA overrides that are currently being applied by navigating to `edge://compat/useragent` in Microsoft Edge Beta.  
+
+Our Canary and Dev channels do not currently receive UA overrides since Microsoft is providing an environment where web developers are easily reproduce issues on their sites that are caused by the default Microsoft Edge UA.  If for some reason you require the ability to disable UA overrides in Microsoft Edge Beta, you may run `msedge.exe` using the following command line argument:  

```

Comment:
Perhaps

"Our Canary and Dev channels do not currently receive UA overrides so that web developers have an environment where they can easily reproduce issues on their sites that are caused by the default Microsoft Edge UA." ?